docker-compose.yml

```diff
@@ -112,12 +112,22 @@ services:
       - POSTGRES_DATABASE_PASSWORD=${POSTGRES_DATABASE_PASSWORD}
       - POSTGRES_DATABASE_HOST=${POSTGRES_DATABASE_HOST}
       - POSTGRES_DATABASE_PORT=${POSTGRES_DATABASE_PORT}
-      - OPENAI_API_TYPE=${OPENAI_API_TYPE}
+      # R2R DB variables
+      - POSTGRES_USER=${POSTGRES_DATABASE_USERNAME}
+      - POSTGRES_PASSWORD=${POSTGRES_DATABASE_PASSWORD}
+      - POSTGRES_HOST=${POSTGRES_DATABASE_HOST}
+      - POSTGRES_PORT=${POSTGRES_DATABASE_PORT}
+      - POSTGRES_DBNAME=${POSTGRES_DATABASE_NAME}
+      # New variables
+      - POSTGRES_VECS_COLLECTION=${POSTGRES_VECS_COLLECTION}
```

Comment:
Collection should not be hardcoded in the ENV variables.

We can have multiple collections. It should be a parameter in the Queue Message.

Reply:
Sure updating it. This can also be set through python by using the input parameter from queue message.